### ISSUE TITLE
All classification results are 0.5s

### ISSUE THREAD

**Issue by bea-tree** [dmlc/xgboost#1088]
Hi, I am new in learning Xgboost, and i have a question like this:
when I try to use xgboost to classify some data ,such as when [0,0,0,0,0,0] label is [1]
when [1 1 1 1 1 1],label is [0],but all the classificaion  resulta are 0.5s.
I know it must be my mistakes when use it, it has maken me confused for 1 week, Thank you for your tiime. ,please help me about this when you are convenience. 
And this is my code:

`import  xgboost as xgb
import numpy as np
from sklearn.cross_validation import train_test_split
from sklearn.metrics import f1_score
random_seed=333
params={
    'booster':'gbtree',
    'objective': 'binary:logistic',
    'early_stopping_rounds':1000,
    'gamma':0.1,
    'max_depth':10,
    'lambda':300,
    'alpha':100,
    'subsample':1,
    'colsample_bytree':1,
    'min_child_weight':5,
    'eta': 0.001,
    'seed':random_seed,
   # 'silent':1
    }

def evalerror(y,t):
    t = t.get_label()
    y_bin = [1 if y_cont > 0.5 else 0 for y_cont in y] # binaryzing your output
    return 'f1',f1_score(t,y_bin)
sample_r_array=[[1,1,1,1,1,1,1,1,1,1,0],\
                [1,1,1,1,1,1,1,1,1,1,0],\
                [1,1,1,1,1,1,1,1,1,1,0],\
                [1,1,1,1,1,1,1,1,1,1,0],\
                [1,1,1,1,1,1,1,1,1,1,0],\
                [1,1,1,1,1,1,1,1,1,1,0],\
                [1,1,1,1,1,1,1,1,1,1,0],\
                [1,1,1,1,1,1,1,1,1,1,0],\
                [1,1,1,1,1,1,1,1,1,1,0],\
                [1,1,1,1,1,1,1,1,1,1,0],\
                [0,0,0,0,0,0,0,0,0,0,1],\
                [0,0,0,0,0,0,0,0,0,0,1],\
                [0,0,0,0,0,0,0,0,0,0,1],\
                [0,0,0,0,0,0,0,0,0,0,1],\
                [0,0,0,0,0,0,0,0,0,0,1],\
                [0,0,0,0,0,0,0,0,0,0,1],\
                [0,0,0,0,0,0,0,0,0,0,1],\
                [0,0,0,0,0,0,0,0,0,0,1],\
                [0,0,0,0,0,0,0,0,0,0,1]]
train_set=np.array(sample_r_array)

test=[1,1,1,1,1,1,1,1,1,1]
X_train,X_val,y_train,y_val=train_test_split\
    (train_set[:,0:-1],train_set[:,-1],test_size=0.3,random_state=random_seed,feval=evalerror)
dval = xgb.DMatrix(X_val,label=y_val)
dtrain = xgb.DMatrix(X_train, label=y_train)
dtest=xgb.DMatrix(test)
watchlist  = [(dval,'eval'), (dtrain,'train')]
model = xgb.train(params,dtrain,num_boost_round=10,evals=watchlist)
a=model.predict(dtest)
print a`

**Comment by khotilov** [comment]
Do not expect xgboost to work with every possible crazy combination of parameters you can throw at it.
Did you even try running it with different parameters? 

The l1 and l2 regularizations are way too high in `params`, and also that min_child_weight might be too large for such a small sample.
